notebooks/Intro to Coordinate Clustering.ipynb

```diff
@@ -53,9 +53,9 @@
     "from sklearn.cluster import DBSCAN\n",
     "\n",
     "import logging\n",
-    "from sensai.util.graph import CoordinateSpanningTree\n",
-    "from sensai.clustering.coordinate_clustering import SKLearnCoordinateClustering\n",
-    "from sensai.util.geometry import alphaShape\n",
+    "from sensai.geoanalytics.graph import CoordinateSpanningTree\n",
```

Comment:
Hi @schroedk, glad to see you working on sensai again :). This PR is a breaking change for us, of course, but I guess it won't be too hard to adjust the imports

Reply:
Hi @MischaPanch , hope changes are not to hard to adjust to :)